azerothcore/azerothcore-wotlk
# Db fix quest nessashadowsong

## Changes Proposed:
- Changed the allowed races from Alliance to Night Elf only for this quest chain

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7693
- Closes https://github.com/chromiecraft/chromiecraft/issues/1601

## SOURCE:


- The chain's starter quest, Nessa Shadowsong, has this comment: "This is one of four starter area Flight Point Quests. You cannot complete this as any other race save Night Elf." https://tbc.wowhead.com/quest=6344/nessa-shadowsong#comments:id=21974
- "This quest is part of the Night elf Taxi! quest chain."  https://wowpedia.fandom.com/wiki/Flight_to_Auberdine?oldid=2178485
- (https://tbc.wowhead.com/quest=6342/flight-to-auberdine)
- https://wotlkdb.com/?quest=6342
- https://classic.wowhead.com/quest=6342/flight-to-auberdine
- https://www.youtube.com/watch?v=7hr7D4l3tIE


## Tests Performed:

- Ran SQL, 4 Affected Rows, 0 Errors
![image](https://user-images.githubusercontent.com/44707108/131909476-06bb8c55-6ba7-47dd-9518-f5c1136a0462.png)

- Tested in-game

https://user-images.githubusercontent.com/44707108/131910362-95036276-5fc4-4d30-8d23-074947e8c60d.mp4

## How to Test the Changes:


1. .quest add 6341 as allied races except night elf
2. .go c 49938
3. see if you can accept to next quest



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
